src/GovUk.Education.ExploreEducationStatistics.Data.Api/Startup.cs

```diff
@@ -143,6 +145,7 @@ public void ConfigureServices(IServiceCollection services)
                         new BlobServiceClient(publicStorageConnectionString),
                         provider.GetRequiredService<ILogger<BlobStorageService>>(),
                         new StorageInstanceCreationUtil()));
+            services.AddTransient<IReleaseFileBlobService, PublicReleaseFileBlobService>();
```

Comment:
Public/private implementations have been configured as services in Admin and Data.Api but look like they are not actually in use yet. Should it be configured in `Content.Api` for consistency?

Reply:
Good point, I've added it to the `Content.Api` startup as well!